CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ Added a new validation (`PA137`) to **validate**, ensuring the absence of non-ig
 * Updated the **format** command to use the content graph instead of the id_set file.
 * Updated **format** command not to fail on unexpected values that returns from the graph, and just add it to the log.
 * Removed a redundant debug log on the `tools.get_file` function.
+* Added a metadata key to the XSIAM dashboard schema.
```

Comment:
pay attention your changelog is on 1.19.0
make sure to add it to `Unreleased` section, you should merge from master

```suggestion
* Added the *metadata* key to the XSIAM dashboard schema. <explain shortly the purpose of this field>
```